README.md

```diff
@@ -51,11 +51,23 @@ es.addEventListener("error", (event) => {
   }
 });
 
+es.addEventListener("done", (event) => {
+  console.log("Done SSE connection.");
+});
+
 es.addEventListener("close", (event) => {
   console.log("Close SSE connection.");
 });
 ```
 
+### Done vs Close
+
+`done` events will fire when server closes the connection.
+Reconnections will occur indefinitely, unless this behavior is disabled.
```

Comment:
I think this should be rephrased slightly to make it more clear. Maybe something like:

By default, the client will automatically reconnect when this happens. You can disable the reconnections by setting the `pollingInterval` option to 0".

Reply:
Sounds good to me. Updated.